src/heretic/doomdef.h

```diff
@@ -563,6 +563,7 @@ extern int prevmap;
 extern int totalkills, totalitems, totalsecret; // for intermission
 extern int levelstarttic;       // gametic at level start
 extern int leveltime;           // tics in game play for par
+extern int totalleveltimes; // [crispy] total time for all completed levels
```

Comment:
I added this line because it was here on the doom side, but it isn't necessary for this PR I think. It could be left out until we add total time to intermissions and such.

Reply:
Let's keep it for this very purpose.

Reply:
Just asking, are you going to work on this?

Reply:
I will be looking into it.